content/millennium/r4/clinical/summary/family-member-history.md

```diff
@@ -15,10 +15,74 @@ The FamilyMemberHistory resource represents family history for a given patient.
 
 The resource should be leveraged as a snapshot in time and new data should be consistently retrieved through the API rather than stored within an application. There may be instances where a relative had an associated condition removed from the patient’s chart which would then no longer be exposed on a subsequent transaction. It is crucial that the data be refreshed with every interaction.
 
+It is recommended that an update to FamilyMemberHistory contain all relevant data that previously existed on the resource to ensure that data remains populated in the patient's chart (e.g., FamilyMemberHistory.sex, FamilyMemberHistory.bornDate, FamilyMemberHistory.condition.onsetDate, etc.). The absence of this optional information may result in the removal of data from the patient's chart.
+
 The following fields are returned if valued:
 
-* [FamilyMemberHistory id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
- 
+* [Id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](http://hl7.org/fhir/r4/familymemberhistory-definitions.html#FamilyMemberHistory.status){:target="_blank"}
```

Comment:
all these need to be https.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/728/commits/9b6b77e5205ad78964a0ac908ff67562395e225f#diff-8e226cf2e473b91b8c3db632af78280b2e61a4509894227d43e038cbd35eee68L23